package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "eslint-plugin-silvermine",
+  "name": "@silvermine/eslint-plugin",
```

Comment:
Sorry for the drive by review, but I saw this in my email and noticed a problem.

ESLint is very particular about how plugins need to be named. The plugin name needs to be in the form `eslint-plugin-{name}`. We can still scope it, but it must be `@{scope}/eslint-plugin-{name}`

Source: https://eslint.org/docs/developer-guide/working-with-plugins (First paragraph)